### PR TITLE
Adding pages and interaction

### PR DESCRIPTION
**Pull request by acabunoc**
* issue a badge form
* added pure css
* added badge & badgeList components
* change DOI endpoint to use two separate doi1/doi2 and not an encoded doi

### CODE REVIEW

**Comment by acabunoc** [comment]
@alicoding, can you r? sorry it's so large...

**Comment by acabunoc** [comment]
This is currently running on http://paperbadger.herokuapp.com/

File: templates/components/badgelist.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var React = require('react'),
+    Badge = require('./badge.jsx'),
+    $ = require('jquery');
+
+var BadgeList = React.createClass({
+  loadUsersFromServer: function() {
+    $.ajax({
```
Comment (alicoding) [inline templates/components/badgelist.jsx]: If you are only using jquery for ajax I'd suggest using **fetch** library (I used that on id.wb.org) or use supwragent instead.

File: templates/pages/home.jsx [hunk 2]
```diff
@@ -1,12 +1,80 @@
...
-      <div>Welcome to PaperBadger</div>
+      <div>
+        <div>
+        <h1>Contributorship Badges</h1>
+        <p>Exploring the use of digital badges for crediting contributors to scholarly papers for their work. <a href="/about">More about contributorship badges.</a></p>
+        </div>
+        <h2>Explore Badges</h2>
+        <ul>
+          <li><a href="/issue">Issue a badge</a></li>
+          <li><a href="/users/0000-0003-4959-3049/badges">Browse issued badges for ORCID 0000-0003-4959-3049</a> (JSON endpoint)</li>
```
Comment (alicoding) [inline templates/pages/home.jsx]: Might be better using <Link/> provided by react router for all internal pages here?...

**Comment by alicoding** [comment]
Left few comments to address

**Comment by acabunoc** [comment]
Thanks! So fast! Will take a look

**Comment by acabunoc** [comment]
Hey @alicoding, made a few more changes. Can you r?

I added isomorphic-fetch, and left the API endpoints as regular links (but updated the internal links to use react-router.

**Comment by alicoding** [comment]
looking good! r+

**Pull request merged**